webpack.config.js

```diff
@@ -186,15 +186,15 @@ export default {
         type: 'asset/resource',
         generator: {
           filename: 'fonts/[name][ext]',
-          publicPath: '/', // required to remove css/ path segment
+          publicPath: '/assets/', // required to remove css/ path segment
```

Comment:
```suggestion
          publicPath: '../', // required to remove css/ path segment
```

Fix from https://github.com/go-gitea/gitea/pull/15501 works here as well.

Reply:
Does it need to be ../assets?

Reply:
No, I tried that before and it gave me a `assets/assets/something` path, so this should be correct, we are modifying the path `css/fonts/something` to `fonts/something`.